system/lib/libc/musl/src/conf/sysconf.c

```diff
@@ -177,9 +180,13 @@ long sysconf(int name)
 	} else if (values[name] >= -1) {
 		return values[name];
 	} else if (values[name] < -256) {
+#ifdef  __EMSCRIPTEN__
+		assert(0);
```

Comment:
at least a comment would be good here? it's not obvious why this is here.

is it intentionally only in debug builds, which means off by default?

but the bigger question is, why is it ok to error here?

Reply:
This is indented like a static assert.    I'll see if I can re-factor to make it more reasonable.

Reply:
Done

Reply:
But why is it ok to avoid the dependency? Did we never support it? (as it is this PR looks like a regression)

Reply:
Interesting question.

After some investigation I found: Before this change this codepath was never being hit because not of values in `values` where < -256.   The `< -256` check is only true for the `RLIM` values above.   So the compilers was completely removing this block avoiding the dependency.  

In an early version of this change I added `RLIM` above which implicitly introduced the dependency.      I think I will revert this `ifdef` not along with the changes to standalone.c

Reply:
This change should be a lot more precise now.